package.json

```diff
@@ -14,6 +14,8 @@
   "keywords": [],
   "author": "Tetlier",
   "dependencies": {
+    "@testing-library/jest-dom": "^5.16.4",
```

Comment:
testing dependencies should be in the devDependencies section instead of main dependencies